source/sandbox.ts

```diff
@@ -116,10 +116,12 @@ async function processEtherdelta(eth: Ethjs) {
 	let sum = new Ethjs.BN(0);
 	const remaining_addresses = {};
 	for (let address in collected_addresses) {
-		const { tokens_escrowed } = await etherdelta.tokens(serpent_rep_contract_address, address);
-		if (tokens_escrowed == 0) continue;
-		sum = sum.add(tokens_escrowed);
-		remaining_addresses[address] = tokens_escrowed.toString(10);
+		if (collected_addresses.hasOwnProperty(address)) {
```

Comment:
Since this isn't being run in a browser, we don't need to worry about some random stupid JS library doing something crazy and modifying an object that we create several lines up.  This means that this line is unnecessary.